src/locales/en.json

```diff
@@ -1,14 +1,22 @@
 {
-  "Catalog":"Catalog",
-  "Orders":"Orders",
-  "Open":"Open",
-  "Packed":"Packed",
-  "Color":"Color",
-  "Size":"Size",
-  "In Stock":"In Stock",
-  "Copy":"Copy",
-  "Search Orders":"Search Orders",
-  "Handover":"Handover",
-  "Ready For Pickup":"Ready For Pickup",
-  "More":"More"
+  "Catalog": "Catalog",
```

Comment:
Arrange texts here in alphabetical order

Reply:
Done